.github/workflows/prepare-release-branch.yml

```diff
@@ -0,0 +1,100 @@
+name: Prepare release branch
+on:
+  workflow_dispatch:
+
+jobs:
+  create-pull-request-against-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Create release branch
+        id: create-release-branch
+        run: |
+          stable_version=$(./scripts/eachdist.py version --mode stable)
+          stable_version=${stable_version//-dev/}
+
+          prerelease_version=$(./scripts/eachdist.py version --mode prerelease)
+          prerelease_version=${prerelease_version//-dev/}
+
+          stable_version_branch_part=$(echo $stable_version | sed -E 's/([0-9]+)\.([0-9]+)\.0/\1.\2.x/')
+          prerelease_version_branch_part=$(echo $prerelease_version | sed -E 's/0\.([0-9]+)b0/0.\1bx/')
+          release_branch_name="release/v$stable_version_branch_part-$prerelease_version_branch_part"
+
+          git push origin HEAD:$release_branch_name
+
+          echo "STABLE_VERSION=$stable_version" >> $GITHUB_ENV
+          echo "PRERELEASE_VERSION=$prerelease_version" >> $GITHUB_ENV
+          echo "RELEASE_BRANCH_NAME=$release_branch_name" >> $GITHUB_ENV
+
+      - name: Update version
+        run: .github/scripts/update-version.sh $STABLE_VERSION $PRERELEASE_VERSION
```

Comment:
Is there a reason why we can't run `./scripts/eachdist.py update_versions --versions stable,prerelease` directly from here instead of having to call a bash script?

Reply:
You could definitely copy the 3-line shell script directly into the yaml. I primarily separated it into a script so that the `yml` files would be more similar across repos that adopt this (since the mechanics of updating versions themselves will vary a lot between repos, but the other parts of the yml do not vary much).

Reply:
I was more referring to calling the python file directly in the `yml` file. The less scripting we can use the better (we want as many things to be as "pythonic" as possible. But not a blocking comment.